doc/api/assert.md

```diff
@@ -286,26 +286,41 @@ assert.fail('a', 'b');
 // AssertionError: 'a' != 'b'
 ```
 
-## assert.ifError(value)
+## assert.ifError(value, message)
 <!-- YAML
 added: v0.1.97
 -->
 * `value` {any}
+* `message` {any}
 
-Throws `value` if `value` is truthy. This is useful when testing the `error`
-argument in callbacks.
-
+Throws an `AssertionError` if `value` is truthy. This is useful when testing the
+`error` argument in callbacks. If the `message` parameter is undefined, a
+default error message is assigned, and `value` is be appended to it.
```

Comment:
I feel this is somewhat contradicting each other. If it should be used in callbacks, there should always either be an error or the result but never both at the same time.